From 7df99f5adbc370a6494cb68890dacff3227b0f04 Mon Sep 17 00:00:00 2001
From: Ben Hutchings <ben@decadent.org.uk>
Date: Fri, 30 Dec 2011 17:58:57 +0100
Subject: [PATCH] aufs: Do not refer to AUFS_NAME in pr_fmt

AUFS_NAME is only defined in aufs_type.h but pr_fmt may be used in
headers included before aufs_type.h.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 fs/aufs/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fs/aufs/Makefile b/fs/aufs/Makefile
index 58ad988..9594b8a 100644
--- a/fs/aufs/Makefile
+++ b/fs/aufs/Makefile
@@ -5,7 +5,7 @@ include ${srctree}/${src}/magic.mk
 # enable pr_debug
 ccflags-y += -DDEBUG
 # sparse doesn't allow spaces
-ccflags-y += -D'pr_fmt(fmt)=AUFS_NAME"\040%s:%d:%s[%d]:\040"fmt,__func__,__LINE__,current->comm,current->pid'
+ccflags-y += -D'pr_fmt(fmt)="aufs\040%s:%d:%s[%d]:\040"fmt,__func__,__LINE__,current->comm,current->pid'
 
 obj-$(CONFIG_AUFS_FS) += aufs.o
 aufs-y := module.o sbinfo.o super.o branch.o xino.o sysaufs.o opts.o \
-- 
1.7.7.3

